src/tools/rust-analyzer/crates/ide/src/hover/tests.rs

```diff
@@ -6945,9 +6945,8 @@ fn hover_feature() {
             Various intrinsics have native MIR operations that they correspond to. Instead of requiring
             backends to implement both the intrinsic and the MIR operation, the `lower_intrinsics` pass
             will convert the calls to the MIR operation. Backends do not need to know about these intrinsics
-            at all. These intrinsics only make sense without a body, and can either be declared as a "rust-intrinsic"
-            or as a `#[rustc_intrinsic]`. The body is never used, as calls to the intrinsic do not exist
-            anymore after MIR analyses.
+            at all. These intrinsics only make sense without a body, and can be as a `#[rustc_intrinsic]`. 
+            The body is never used, as calls to the intrinsic do not exist anymore after MIR analyses.
```

Comment:
@rust-lang/rust-analyzer is this an RA test having a fully copy of the docs of an unstable feature inside of it? IMO that's a bad idea, it creates unexpected friction when improving those docs. Is there a way to have this test in a way that causes less of a roadblock for unrelated PRs?

Reply:
Yes, it tests that if you hover over a the name of a feature in `#![feature(foo)]` that rust-analyzer presents the documentation rustc has for this feature.

Reply:
That could be tested without requiring a full copy of the docs to be kept in sync. Just check that the string we get is non-empty or contains "the" or something like that.